rust/feature-flags/src/flags/flag_operations.rs

```diff
@@ -55,6 +55,12 @@ impl FeatureFlagList {
         client: Arc<dyn RedisClient + Send + Sync>,
         project_id: i64,
     ) -> Result<FeatureFlagList, FlagError> {
+        tracing::info!(
```

Comment:
I can pass in the rust log level at my argocd deployment, so I don't need to worry about it here.